libs/utils/executor.py

```diff
@@ -341,10 +333,9 @@ def _wload_cpus(self, wl_idx, wlspec):
             if 'last' in cpus:
                 return [ self.target.bl.bigs_online[-1] ]
             return self.target.bl.bigs_online
-        raise ValueError('Configuration error - '
-                'unsupported [{}] \'cpus\' value for [{}] '\
-                'workload specification'\
-                .format(cpus, wl_idx))
+        raise ValueError('unsupported [{}] "cpus" value for [{}] '
+                         'workload specification'
+                         .format(cpus, wl_idx))
```

Comment:
Not sure this hunk is necessary. If it is, it should be in a separate commit. Same for the other ValueError messages below.